backend/lib/services/cloudprofiles.js

```diff
@@ -18,6 +18,7 @@
 
 const { NotFound } = require('../errors')
 const logger = require('../logger')
+const shoots = require('./shoots')
```

Comment:
The api of the services modules is designed to be used from the routes directly. We should be careful with cross service reuse because of cycling dependancies.